jetty-security/src/main/java/org/eclipse/jetty/security/HashLoginService.java

```diff
@@ -187,16 +195,18 @@ protected UserPrincipal loadUserInfo(String userName)
     protected void doStart() throws Exception
     {
         super.doStart();
-        
-        if (_propertyUserStore == null)
+
+        // can be null so we switch to previous behaviour using PropertyUserStore
+        if (_userStore == null)
         {
             if(LOG.isDebugEnabled())
                 LOG.debug("doStart: Starting new PropertyUserStore. PropertiesFile: " + _configFile + " hotReload: " + hotReload);
-            
-            _propertyUserStore = new PropertyUserStore();
-            _propertyUserStore.setHotReload(hotReload);
-            _propertyUserStore.setConfigPath(_configFile);
-            _propertyUserStore.start();
+
+            PropertyUserStore propertyUserStore = new PropertyUserStore();
+            propertyUserStore.setHotReload(hotReload);
+            propertyUserStore.setConfigPath(_configFile);
+            propertyUserStore.start();
+            this._userStore = propertyUserStore;
```

Comment:
Teeny quibble: you can get rid of the this., as the underscore on the field name is definitive.